hosts/OpenIris/home/jalemi/home.nix

```diff
@@ -27,6 +27,7 @@
 
         # Gaming
         bottles
+        osu-lazer
```

Comment:
I recommend not having osu-lazer from NixOS packages, because if you get osu-lazer, it will compile from scratch, if you get osu-lazer-bin, it will be outdated and have slow updates. Please resort to using the flatpak instead if you want to play osu, or use a distrobox container to play with using the latest AppImage as soon as it's out.

Reply:
I appreciate your participating in this PR, so I'm changing to flatpak version, thank you for noticing disadvantages of osu-lazer from nixpkgs.